package.json

```diff
@@ -36,26 +36,31 @@
     "eslint": "eslint .",
     "test": "mocha",
     "jsdoc": "jsdoc lib/* -d dist/doc",
-    "travis": "npm run lint && npm run test"
+    "travis": "npm run lint && npm run test",
+    "browserify": "mkdirp dist && node_modules/browserify/bin/cmd.js lib/index.js -o dist/pagexray.js --standalone PageXray -p [ browserify-banner --template 'PageXray v<%= pkg.version %> - Xray your HAR file and know all about the page - https://github.com/sitespeedio/pagexray' ]",
```

Comment:
This depends on the mkdirp package being installed (which it is via some dependency of a dependency), so we should add that to devDependencies as an explicit dependency.